src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -5,40 +5,35 @@
 
 public class SalaryInfo {
 
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
Constant variables should be all uppercase letters.
You can have a look here: https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names